packages/mui-styled-engine-sc/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/styled-engine-sc",
-  "version": "6.0.0-alpha.18",
+  "version": "6.0.0-alpha.19",
```

Comment:
It's a bit strange that this package version is ahead 😕 , but we started the alpha channel when we were upgrading to support styled-components v6.

Reply:
Oh, it has been a while since https://github.com/mui/material-ui/pull/39374/files#diff-45c30a19b13bd33a9e6d3a4ff2731ac00f357360fe5b13d344ab91c727916fe5